CHANGES.rst

```diff
@@ -4,8 +4,9 @@ History
 2.5 (unreleased)
 ----------------
 
-- Nothing changed yet.
-
+- Provide a way to pass a mrbob configuration file
+ ``
```

Comment:
are these two backticks here by intend or accident?

Reply:
It's just an accident.